.github/workflows/build-deploy.yml

```
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3.3.0
+        uses: actions/checkout@v3.5.3
```

Tror nesten vi bare kan endre til 3, så trenger vi ikke å oppdatere den med minor endringer 👀 
```suggestion
        uses: actions/checkout@v3
```

---

Det visste jeg ikke, kjekt! :D